packages/x-date-pickers/src/internals/hooks/useField/useField.ts

```diff
@@ -101,6 +109,201 @@ export const useField = <
     setSelectedSections(null);
   });
 
+  const handleInputPaste = useEventCallback((event: React.ClipboardEvent<HTMLInputElement>) => {
```

Comment:
I'm adding an error because letting `onChange` handle it would not work.
Implementing it will be an interesting follow up.